app/serializers/articles_show_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class ArticlesShowSerializer < ActiveModel::Serializer
+  attributes :id, :title, :teaser, :date, :body
```

Comment:
Previously you have sent the date after all the content, why are you not keeping the same pattern?